tests/Resources/ti.ui.view.test.js

```diff
@@ -1125,6 +1125,37 @@ describe('Titanium.UI.View', function () {
 			win.add(outerView);
 			win.open();
 		});
+
+		it('1 value to create circle', finish => {
+			win = Ti.UI.createWindow({ backgroundColor: 'blue' });
+			const outerView = Ti.UI.createView({
+				width: '90px',
+				height: '90px',
+				backgroundColor: 'green'
+			});
+			const view = Ti.UI.createView({
+				width: '60px',
+				height: '60px',
+				borderRadius: '30px',
+				backgroundColor: 'yellow'
+			});
+
+			win.addEventListener('postlayout', function postlayout() {
+				win.removeEventListener('postlayout', postlayout); // only run once
+				try {
+					should(view.borderRadius).be.a.String();
+					should(view.borderRadius).eql('30px');
+					should(outerView).matchImage(`snapshots/borderRadius30px_${density}x.png`);
```

Comment:
I just realized - this image should not differ by density, because it's all declared in px, so density here shouldn't matter. Density matters for images when the size of any part is related to the density (i.e. something is using `dp`, `dpi`, `pt` units). So you shouldn't need to inject the density into the filename, and should only need one version of the image per platform.

Reply:
You are right. Updated!